roc/version.go

```diff
@@ -72,3 +73,40 @@ func parseVersion(s string) SemanticVersion {
 
 	return v
 }
+
+func stringifyVersion(sv SemanticVersion) string {
+	return fmt.Sprintf("%d.%d.%d", sv.Major, sv.Minor, sv.Patch)
+}
```

Comment:
nit: instead of stringifyVersion, we can add SemanticVersion.String() method, I think it would be nicer, and versions will automatically support `%s` formatting both for us and for users.